core/cogs/battle.py

```
@@ -51,10 +51,10 @@ async def cog_load(self):
                 if self.settings.shortform
                 else self.bot.alias["battle"]["normal"]
             )
-            asyncio.create_task(self.bot.put_queue(self.cmd))
+            asyncio.create_task(self.bot.ch.put_queue(self.cmd))
```

Ruff (0.16.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)